CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 ### Added
 
 - Use validation logic from the app library.
+- Update appcatalogentry CRs if it contains restrictions on metadata.
```

Comment:
```suggestion
- Include restrictions data from app metadata files in appcatalogentry CRs.
```

What about this? 

I think it's important to mention we're fetching the app metadata files. Soon we'll include more types of data.